app/helpers/event_front_end.rb

```diff
@@ -0,0 +1,5 @@
+module EventFrontEnd
```

Comment:
http://nithinbekal.com/posts/rails-presenters/

I believe this should be a presenter. 

Helpers can be useful for cleaning up view logic, but rails includes every helper in every view context by default